src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    protected String by;
```

Comment:
do you think maybe a noun will be a better variable name here? maybe something like "dueDate"?